### PR TITLE
Refactor command execution and implement concurrent orchestrator and API

### PR DESCRIPTION
**Pull request by iuwqyir**
### TL;DR

Refactored command structure and implemented concurrent execution of orchestrator and API.

### What changed?

- Extracted the main logic from `api` and `orchestrator` commands into separate functions (`RunApi` and `RunOrchestrator`).
- Updated the root command to run both the orchestrator and API concurrently.
- Removed the placeholder log message in the root command.

### How to test?

1. Run the root command to ensure both the orchestrator and API start concurrently.
2. Test the `api` and `orchestrator` commands individually to verify they still function as expected.
3. Check logs to confirm that both services are starting up correctly.

### CODE REVIEW

**Comment by iuwqyir** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/indexer/42?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#42** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/indexer/42?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#41** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/indexer/41?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iuwqyir and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .dockerignore [hunk 1]
```diff
@@ -1 +1,4 @@
-.env
\ No newline at end of file
+.env
+
+configs/secrets*
+!configs/secrets.example*
```
Comment (AmineAfia) [inline .dockerignore:4]: let's be explicit about secrets files

```suggestion
!configs/secrets.example
```

**Review by AmineAfia — APPROVED** [review]

File: .dockerignore (hunk 1 above)
Comment (iuwqyir) [reply .dockerignore:4]: we can actually ignore everything in the docker container, it doesn't need the example file

Comment (iuwqyir) [reply .dockerignore]: changed

**Review by AmineAfia — APPROVED** [review]

**Comment by iuwqyir** [comment]
### Merge activity

* **Sep 25, 6:48 PM EDT**: @iuwqyir started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thirdweb-dev/indexer/42).
